From: Paul Mundt <lethal@linux-sh.org>
Date: Wed, 28 Oct 2009 18:02:15 +0900
Subject: [PATCH 003/279] sh: perf events: Kill off left over debugging cruft.
Git-commit: 1d317f90d97ca8e539939ee896bd04c7efe936ca
Patch-mainline: v2.6.33-rc1
References: FATE#311392, BNC#685313
Signed-off-by: Tony Jones <tonyj@suse.de>

num_events should be compared > MAX_HWEVENTS and not >=. The latter was
used as a debugging test which accidentally slipped in.

Signed-off-by: Paul Mundt <lethal@linux-sh.org>
Signed-off-by: Robert Richter <robert.richter@amd.com>
---
 arch/sh/kernel/perf_event.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/arch/sh/kernel/perf_event.c b/arch/sh/kernel/perf_event.c
index d151070..4449f0a 100644
--- a/arch/sh/kernel/perf_event.c
+++ b/arch/sh/kernel/perf_event.c
@@ -308,7 +308,7 @@ int register_sh_pmu(struct sh_pmu *pmu)
 
 	pr_info("Performance Events: %s support registered\n", pmu->name);
 
-	WARN_ON(pmu->num_events >= MAX_HWEVENTS);
+	WARN_ON(pmu->num_events > MAX_HWEVENTS);
 
 	return 0;
 }
-- 
1.7.3.4

